src/components/Header/Header.tsx

```diff
@@ -0,0 +1,62 @@
+import { Pages, Paths } from '../../constants/common.constants';
+import React from 'react';
```

Comment:
no neeeeed, remo React import rom all components))

Reply:
I am using class components. I need it)